libretroshare/src/jsonapi/jsonapi.cpp

```diff
@@ -41,6 +41,26 @@
 
 /*extern*/ JsonApiServer* jsonApiServer = nullptr;
 
+/*static*/ const std::multimap<std::string, std::string>
+JsonApiServer::corsHeaders =
+{
+    { "Access-Control-Allow-Origin", "*" },
```

Comment:
No, it doesn't mean that, it means that if you access the API trough a browser it doesn't mind the domain name you have on your urlbar, so for example if an APP based on electron or other web framework uses RS API it could access even if they define their own alias of 127.0.0.1 to be more user friendly or use another port fro their own HTTP server on same machine. RetroShare JSON API bind on 127.0.0.1:9092 if it is not otherwise specified by command-line options, or by RetroShare GUI options.